pkg/provider/azure_loadbalancer_backendpool.go

```diff
@@ -792,6 +796,23 @@ func newBackendPool(lb *network.LoadBalancer, isBackendPoolPreConfigured bool, p
 }
 
 func (az *Cloud) addNodeIPAddressesToBackendPool(backendPool *network.BackendAddressPool, nodeIPAddresses []string) bool {
```

Comment:
This func is used in both cluster and local svc reconciliations, so move the vnet code to here.